src/main/java/org/myrobotlab/programab/Oob.java

```diff
@@ -0,0 +1,14 @@
+package org.myrobotlab.programab;
+
+public class Oob {
```

Comment:
why didn't you use OOBPayload?  ..

Reply:
:shrug: no direction given

Reply:
I could switch, or create a new definition - but the whole Sraix handling needs some design

Reply:
agreed.. i want to get through the rest of the review on this PR...  i might want to pull the programAB stuff into a different PR...

Reply:
sounds good :+1: